docs/release-notes/cli-uuid-prefixes.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
```

Comment:
Sorry for the delay.
This could use more context, such as describing the problem this feature solved, from the DET-6706 description field.
Also, is this really a 'new feature' or more of a 'fixed'?

Reply:
Okay, I added more context. I'd say it's a new feature; it's not incorrect for the CLI to require a full UUID, just not the most convenient possible functionality.